src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/AnchorLayoutTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Drawing;
+using System.Windows.Forms.Layout;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Windows.Forms.UITests
+{
+    public class AnchorLayoutTests : ControlTestBase
+    {
+        public AnchorLayoutTests(ITestOutputHelper testOutputHelper)
+            : base(testOutputHelper)
+        {
+        }
+
+        private static (Form, Button) GetFormWithAnchoredButton()
+        {
+            Form form = new();
+            form.Size = new Size(200, 300);
+            Button button = new();
+            button.Location = new Point(20, 30);
+            button.Size = new Size(20, 30);
+            button.Anchor = AnchorStyles.Left | AnchorStyles.Top | AnchorStyles.Right | AnchorStyles.Bottom;
+
+            return (form, button);
+        }
+
+        [WinFormsTheory]
+        [InlineData(AnchorStyles.Top, 120, 30, 20, 30)]
+        [InlineData(AnchorStyles.Left, 20, 180, 20, 30)]
+        [InlineData(AnchorStyles.Right, 220, 180, 20, 30)]
+        [InlineData(AnchorStyles.Bottom, 120, 330, 20, 30)]
+        [InlineData(AnchorStyles.Top | AnchorStyles.Left, 20, 30, 20, 30)]
+        [InlineData(AnchorStyles.Top | AnchorStyles.Right, 220, 30, 20, 30)]
+        [InlineData(AnchorStyles.Top | AnchorStyles.Bottom, 120, 30, 20, 330)]
+        [InlineData(AnchorStyles.Left | AnchorStyles.Right, 20, 180, 220, 30)]
+        [InlineData(AnchorStyles.Bottom | AnchorStyles.Right | AnchorStyles.Top | AnchorStyles.Left, 20, 30, 220, 330)]
+        public void ResizeAnchoredControlsParent_HanldeCreated_AnchorsApplied(AnchorStyles anchor, int x, int y, int width, int height)
```

Comment:
```suggestion
        public void ResizeAnchoredControlsParent_HanldeCreated_AnchorsApplied(AnchorStyles anchor, int expectedX, int expectedY, int expectedWidth, int expectedHeight)
```

Any reason not to use `Rectangle` instead?

Reply:
Each anchor property is a separate calculation and like to keep it that way in tests.